### ISSUE TITLE
"New Game" button doesn't clear out added Bless/Curse cards

### ISSUE THREAD

**Issue by kiwitrav** [orichalque/yet-an-other-gloomhaven-companion#226]
Gidday! Something I've noticed a time or two, though because it can be worked around by simply running through the modifier deck and then clicking New Game it hasn't particularly bothered me.

Steps to replicate:

Add a bless or curse (or both)

Click "New Game"

Bless and/or curse still in modifier deck - visible on the "bless" & "curse" deck-controls and also visible using "sort modifiers" functionality as well as under "Modifiers" page.

If you "save" with the bless/curse, it persists across sessions.

Expected behaviour:
"New Game" clears bless/curse
"Save" does not persist bless/curse

Thanks again for this awesome platform!

**Comment by orichalque** [comment]
Hi, thanks for opening this issue.
Thats annoying, you can still remove them through the modifier menu I guess but that's only a workaround.

**Comment by kiwitrav** [comment]
Huh, just noticed "New Game" doesn't deselect selected cards either.

I'm going to suggest ignoring the "Save" comments above as it seems consistent that it saves game in current state, including current turn, selected cards, etc - it's consistent, and while not what I was *expecting* is still a perfectly valid functionality.

**Comment by orichalque** [comment]
Now that you mention it, I remember that there's a "Save game state" option. Maybe you can try disabling it 😁

**Comment by kiwitrav** [comment]
Hah, true - that clears the selected cards between sessions, but (probably because they become part of the modifier deck I'm guessing?) doesn't clear the bless/curse.

**Comment by orichalque** [comment]
I finally implemented a fix.
We discussed a bit about this issue with @RaphaelPage0110 , and came to the conclusion that we may not *always* clear the curses/blessings on new game: if you donated to the great oak sanctuary and lost the game, you would retry and still keep your blessings. 
Thus, I added a small dialog that only opens if you press "new game" and still have blessings/curses in your deck, and that let you choose to remove curses, or not.
